sdk/src/androidTest/java/co/omise/android/ParcelableTest.kt

```diff
@@ -0,0 +1,96 @@
+package co.omise.android
+
+import android.os.Bundle
+import android.os.Parcel
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import co.omise.android.models.*
+import org.joda.time.DateTime
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ParcelableTest {
```

Comment:
I believe this test class can run in unit test. Could you check it?

Reply:
No it cannot as `Parcel` is part of the Android sdk

Reply:
Are you sure? Since we setup Robolectric that can include all Android resources into Unit test. I think every thing that you can run with `AndroidJUnit4` it should runnable in unit test. 🤔

Reply:
We have? When did that happen? I have zero experience with Roboelectric, so please let me know how this can be changed into a unit test 👍

Reply:
Done 👌